pages/index.js

```diff
@@ -212,16 +215,14 @@ export default function Home() {
   async function uploadPhoto(e) {
     const file = e.target.files[0];
     try {
-      await Storage.put(currentContact.id, file, {
+      const result = await Storage.put(currentContact.id, file, {
         contentType: "image/png",
```

Comment:
I don't think you should set the contentType, because a user may not be uploading a png and this would then be wrong.

Reply:
This line has been removed.